src/Pages/CollectionPage.js

```diff
@@ -17,57 +17,67 @@ const base = new Airtable({ apiKey: airtableConfig.apiKey })
   .base(airtableConfig.baseKey);
 
 function CollectionPage() {
-  const [CollectionDetails, setCollectionDetails] = useState();
+  const [CollectionDetails, setCollectionDetails] = useState(null);
   const params = useParams();
-  const [collecID, setCollecID] = useState(params.id);
-  const [collections, setCollections] = useState([]);
+  const [collecID, setCollecID] = useState(null);
+  const [collections, setCollections] = useState(null);
   const getCollections = () => {
     base('Collection').select({ view: 'Grid view' }).all() // Gets + returns all records
       .then((records) => { // Takes in returned records + calls setPosts to store in posts arr
         setCollections(records);
+        setCollecID((params.id === 'init' ? records[0].id : params.id));
       });
   };
   const getPosts = () => {
-    base('Collection').find(
-      collecID,
-      (err, record) => {
-        setCollectionDetails(record);
-      },
-    );
+    if (collecID !== null && collecID !== 'init') {
+      base('Collection').find(
+        collecID,
+        (err, record) => {
+          if (err) { console.error(err); }
+          setCollectionDetails(record);
+        },
+      );
+    }
   };
 
   const updateCollecID = useCallback((newValue) => setCollecID(newValue), [setCollecID]);
 
   useEffect(getPosts, [collecID]);
-  useEffect(getCollections, []);
+  useEffect(() => {
```

Comment:
Note for the future: Your current error-checking method means that the errors are shown even if the content just hasn't fully been loaded. Instead it would be better to have loading screens for the loading states (designers are working on this rn) and error screens for the specific errors. This will have to wait till the designs are ready though!